src/components/SarStakingWidget/AddStake/index.tsx

```diff
@@ -0,0 +1,241 @@
+              height="46px"
+            >
+              {t('earn.approve')}
+            </Button>
+          )}
+          <Button
+            variant={'primary'}
+            isDisabled={!selectedPosition || !!error || approval !== ApprovalState.APPROVED}
+            onClick={() => setOpenDrawer(true)}
+            height="46px"
+          >
+            {!selectedPosition ? t('sarStakeMore.choosePosition') : error ?? t('sarStakeMore.add')}
+          </Button>
+        </Buttons>
+      );
+    }
+  };
+
+  const ConfirmContent = (
+    <Wrapper paddingX="20px" paddingBottom="20px">
+      <Header>
+        <TokenRow>
+          <Text fontSize={24} fontWeight={500} color="text1" style={{ marginRight: '12px' }}>
+            {parsedAmount?.toSignificant(6) ?? 0}
+          </Text>
+          <CurrencyLogo currency={png} size={24} imageSize={48} />
+        </TokenRow>
+        <Box display="inline-grid" style={{ gridGap: '10px', gridTemplateColumns: 'auto auto' }}>
+          <Stat
+            title={t('sarStake.dollarValue')}
+            titlePosition="top"
+            stat={`${dollerWorth ?? 0}$`}
+            titleColor="text2"
+          />
+          <Stat title={t('sarStakeMore.newAPR')} titlePosition="top" stat={`${newAPR}%`} titleColor="text2" />
+        </Box>
+        <Box display="flex" flexDirection="row" justifyContent="space-between">
+          <Text color="text1">{t('sarStake.weeklyDistributed', { symbol: png.symbol })}</Text>
+          <Text color="text1">{numeral(formatEther(weeklyPNG ?? 0)).format('0.00a')}</Text>
+        </Box>
+        <Text color="text1" fontWeight={400} fontSize="14px" textAlign="center">
+          {t('sarStake.confirmDescription')}
+          <br />
+          <br />
+          {t('sarStakeMore.confirmDescription')}
+        </Text>
+      </Header>
+      <Footer>
+        <Box my={'10px'}>
+          <Button variant="primary" onClick={onStake}>
+            {t('sarStakeMore.add')}
+          </Button>
+        </Box>
+      </Footer>
+    </Wrapper>
+  );
+
+  return (
+    <Box>
+      <Root>
+        <Title selectedOption={selectedOption} onChange={onChange} />
+        <Box>
+          <Box justifyContent="space-between" display="flex">
+            <Text color="text1" fontSize="18px" fontWeight={500}>
+              {t('sarStakeMore.stakeMore')}
+            </Text>
+            <Text color="text4">
+              {t('sarStake.walletBalance', { symbol: png.symbol, balance: userPngBalance?.toSignificant(2) ?? 0 })}
+            </Text>
+          </Box>
+          <TextInput
+            value={typedValue}
+            isNumeric={true}
+            placeholder="0.00"
+            onChange={(value: any) => {
+              onUserInput(value);
+            }}
+            addonAfter={
+              <Button variant="plain" backgroundColor="color2" padding="10px" onClick={handleMax}>
+                <Text color="text1">MAX</Text>
+              </Button>
+            }
```

Comment:
Inputs are looking too big in height. Please match it with rest of the website. 
I think its due to Button height. if needed please reduce button height or padding.

![Screenshot 2022-08-09 at 5 16 26 PM](https://user-images.githubusercontent.com/6604146/183639614-57bbfbbe-f6b2-4e51-ae86-c1c2e7b1f300.png)